Coupled-Electron-Pair-Approximation/DF-LCCD.py

```diff
@@ -0,0 +1,79 @@
+"""
+Computes the Linearized CCD AKA CEPA(0) without singles, correlation energy.
+Equations taken by linearizing and density fitting Eq. 153 of [Crawford:2000:33].
+
+__authors__   =  "Jonathon P. Misiewicz"
+__credits__   =  ["Jonathon P. Misiewicz"]
+
+__copyright__ = "(c) 2014-2020, The Psi4NumPy Developers"
+__license__   = "BSD-3-Clause"
+"""
+
+import numpy as np
+import psi4
+from DSD import DirectSumDiis
+from integrals import integrals_DF
+
+### Settings
+
+mol = psi4.geometry("""
+O
+H 1 1.0
+H 1 1.0 2 104.5
+symmetry c1""")
+scf_type = "df"
+target_convergence = 7
+freeze_core = True
+maxiter = 50
+compare_psi4 = True
+basis = "cc-pvdz"
+
+### Setup
+psi4.set_options({"freeze_core": freeze_core, "scf_type": scf_type, "e_convergence": target_convergence + 1, "basis": basis, "cc_type": "df"})
+R, F = integrals_DF(mol)
+Fo = F["oo"].diagonal()
+Fv = F["vv"].diagonal()
+D = Fo.reshape(-1, 1, 1, 1) + Fo.reshape(-1, 1, 1) - Fv.reshape(-1, 1) - Fv
+t2 = np.zeros(D.shape)
+dsd = DirectSumDiis(3, 8)
+nvir = len(Fv)
+
+### Main Loop
+for i in range(1, maxiter + 1):
+    # Two Electron Terms
+    # r2 = g^ij_ab
+    temp = np.einsum("iaq, jbq -> ijab", R["ov"], R["ov"], optimize = True)
```

Comment:
`optimize=True`, can you run a formatter over this in general?

Please do leave spaces above comment lines as well.

Reply:
As in, every comment should have a blank line before it?

Reply:
Yes, help readability I believe.

Reply:
Formatted with column limit 119.